pkg/pinentry/pinentry.go

```diff
@@ -7,12 +7,26 @@ import (
 	"bytes"
 	"fmt"
 	"io"
+	"net/url"
+	"os"
 	"os/exec"
 	"strings"
 
 	"github.com/pkg/errors"
 )
 
+var (
+	// Enable unescaping of percent encoded values, disabled by default
```

Comment:
comment on exported var Unescape should be of the form "Unescape ..."

Reply:
Done